contracts/modules/LoanOpenings/LoanOpenings.sol

```diff
@@ -13,55 +13,42 @@ import "../../mixins/InterestHandler.sol";
 import "../../swaps/SwapsUser.sol";
 import "../../governance/PausableGuardian.sol";
 
-
 contract LoanOpenings is State, LoanOpeningsEvents, VaultController, InterestHandler, SwapsUser, PausableGuardian {
-
-    function initialize(
-        address target)
-        external
-        onlyOwner
-    {
+    function initialize(address target) external onlyOwner {
         _setTarget(this.borrowOrTradeFromPool.selector, target);
         _setTarget(this.setDelegatedManager.selector, target);
+        _setTarget(this.generateLoanParamId.selector, target);
+        _setTarget(this.getDefaultLoanParams.selector, target);
         _setTarget(this.getRequiredCollateral.selector, target);
-        _setTarget(this.getRequiredCollateralByParams.selector, target);
```

Comment:
shall we set these to 0 address on target?

Reply:
its a view only function. shall we really?

Reply:
it just cleans it up a bit. Not necessary but good to only keep up-to-date functions in the proxy.